lld/test/wasm/Inputs/signature-mismatch-debug-info-a.ll

```diff
@@ -0,0 +1,31 @@
+target triple = "wasm32-unknown-emscripten"
+
+define void @foo(i32 %a) !dbg !6 {
+  ret void
+}
+
+define void @test0() !dbg !10 {
+entry:
```

Comment:
Why does this function have `entry:` but not the one above?

Reply:
I generated these tests from `.c` file and manually removed some unnecessary parts, and I think I left some `entry`s. Removed them for consistency.